paddle/fluid/operators/gaussian_random_op.cc

```diff
@@ -29,8 +29,14 @@ class CPUGaussianRandomKernel : public framework::OpKernel<T> {
     float mean = context.Attr<float>("mean");
     float std = context.Attr<float>("std");
     auto* tensor = context.Output<framework::Tensor>("Out");
+    if (context.HasInput("Shape")) {
+      const framework::Tensor* shapeTensor =
+          context.Input<framework::Tensor>("Shape");
+      const int64_t* shapeData = shapeTensor->data<int64_t>();
+      std::vector<int64_t> shape(shapeData, shapeData + shapeTensor->numel());
```

Comment:
Note code style:   shape_tensor,  shape_data

https://google.github.io/styleguide/cppguide.html#Variable_Names

Need to modify all following codes